frontend/types.d.ts

```diff
@@ -0,0 +1,61 @@
+declare global {
+    /**
+     * This namespace is meant for [PlutoDesktop](https://github.com/JuliaPluto/PlutoDesktop)
+     * related types and interfaces.
+     */
+    namespace Desktop {
+        /**
+         * This type has to be in sync with the type "Channels"
+         * defined in PlutoDesktop/{branch:master}/src/main/proload.ts
+         */
+        type Channels = "ipc-example"
```

Comment:
Could this be changed to something more specific to its function?

```suggestion
        type Channels = "ipc-plutodesktop"
```

Reply:
I mean I can definitely do it, and the fact is that this channel is not actually needed. I just made it so that if anyone wants to use `sendMessage` they can modify the `Channels` to make it type-safe.
Should I remove it?

Reply:
If it's not used right now I'd say remove it. In the future if someone needs ipc they can add the channel type back in

Reply:
done!